### DISCUSSION TITLE
请问一下怎么自定义词库

### DISCUSSION THREAD

**Discussion by x0710** [KyleBing/rime-wubi86-jidian#145] [Q&A] [answered]
我在./wubi86_jidian_user.dict.yaml中在下面添加了一些自定义词组，重新部署后仍然无法打出这些词组，已经使用tab，请问还需要做些什么吗？
`
 18 ...
 19 ## 这是分组名
 20 分组词条  wxyt  1
 21 哈哈哈哈哈哈  kkkk  1
 22 还行  gitf  1
 23 
`

**Comment by ZmdnaHF0ZmFqYXZ0O** [comment, marked as answer]
建议参考：wubi86_jidian.dict.yaml 中的格式，每行为一个词组，例如：
```
为人民 ywna
为人民服务   ywnt
```
中文与英文之间的间隔为tab，非空格，供参考.

**Reply by x0710** [reply]
已经解决，默认情况下vim使用空格代替tab
